Mathlib/Analysis/SpecialFunctions/Log/Basic.lean

```diff
@@ -417,6 +417,12 @@ theorem isLittleO_log_id_atTop : log =o[atTop] id :=
   isLittleO_pow_log_id_atTop.congr_left fun _ => pow_one _
 #align real.is_o_log_id_at_top Real.isLittleO_log_id_atTop
 
+theorem isLittleO_const_log_atTop {c : ℝ} (hc : c ≠ 0) : (fun _ => c) =o[atTop] log := by
```

Comment:
You don't need `c ≠ 0` here, see `isLittleO_const_left`.

Reply:
Indeed, not sure why I even had it there.